Configuration/Geometry/python/GeometryExtended2017Reco_cff.py

```diff
@@ -21,7 +21,7 @@
 from Geometry.TrackerGeometryBuilder.idealForDigiTrackerGeometry_cff import *
 from Geometry.CSCGeometryBuilder.idealForDigiCscGeometry_cff import *
 from Geometry.DTGeometryBuilder.idealForDigiDtGeometry_cff import *
-trackerGeometry.applyAlignment = cms.bool(False)
+trackerGeometry.applyAlignment = cms.bool(True)
```

Comment:
May I suggest to just remove this override instead (I think it would be more clear)? The default is `True` https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/Geometry/TrackerGeometryBuilder/plugins/TrackerDigiGeometryESModule.cc#L64.